data/card_catalogue/card-manipulation.json

```diff
@@ -155,19 +142,6 @@
       "image": "res/art/illustration/Tactical Reload.png"
     }
   },
-  {
-    "title": "Cowboy Reload",
-    "upgrade": "Expert Reload",
-    "tier": "red",
-    "details": {
-      "rules": {
-        "active": "Return a red you've cast to your hand.",
-        "passive": null
-      },
-      "flavour": "Cowboys only load up 5 of the 6 bullet chambers. In the 6th chamber, they leave a hundred dollar bill to pay the undertaker.",
-      "image": "res/art/illustration/Cowboy Reload.png"
-    }
-  },
```

Comment:
Same problem as master mimic. It's an incredibly strong card that makes your strategy much more consistent, or does nothing. I don't consider either of those outcomes to be particularly good. Cards should have some kind of tradeoff which would make you want to possibly trade them for other cards. With this line, they're too strong to reasonably discard since unless nobody owns a card of this rarity they're at least as strong as other cards in the same rarity.

I think we should probably get rid of the entire line